### ISSUE TITLE
Prusaslicer 2.6.0 alpha4 exports a gcode file instead of mcfx

### ISSUE THREAD

**Issue by dburden42** [tomvandeneede/p2pp#158]
I'm trying to implement P2PP with the Prusaslicer 2.6.0 alpha4, but exporting gcode doesn't generate a .mcfx file (even though it has the .mcfx extension). 

Here's two files that I exported from the same project, one from Prusaslicer 2.5.0, and one from Prusaslicer 2.6.0 alpha4
![file types](https://user-images.githubusercontent.com/126613706/229870671-6d8667fa-026f-4459-a89e-5d80349c8cb5.jpg)

**Comment by tomvandeneede** [comment]
Deos the p2pp screen popup in 2.6.0?

**Comment by dburden42** [comment]
Yes, the screen pops up and seems to have the same content regardless of which version of Prusaslicer I am using.

**Comment by Killerskyman** [comment]
looks like you don't have view file extensions enabled. 
If you look at the 'type' column, file explorer says that top one is gcode. Also the file icons (left side) are different even though it looks like those files have the same extension. File explorer bases both of those off of the file extension that is there. I think that bottom *.mcfx file is actually named as *.mcfx.mcfx and the top is named *.mcfx.gcode. prusa 2.6.0 seems to force the use of .gcode at the end of the export name.
